### PR TITLE
Accessibillity

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,12 +22,18 @@ <h1 class="title-header">Welcome to Acme Web Sulutions</h1>
           consectetur, adipisci velit...
         </div>
         <div class="button-container">
-          <button class="button-header">
+          <button
+            aria-label="jump to seconed section"
+            tabindex="0"
+            role="button"
```
Comment (takrami) [inline index.html]: is it necessary to put role = button here? doesn't the button tag have it by default? or am i missing sth?

File: index.html [hunk 2]
```diff
@@ -52,7 +58,7 @@ <h2 class="first-section-title">Web & Application Development</h2>
     <section id="jump-seconed-section" class="second-section">
       <ul class="column-container">
         <li class="column-item">
-          <img class="responsive" src="./assets/img/1.jpg" />
+          <img class="responsive" src="./assets/img/1.jpg" alt="laptop" />
```
Comment (takrami) [inline index.html:61]: nice job with alts :)

File: index.html [hunk 3]
```diff
@@ -22,12 +22,18 @@ <h1 class="title-header">Welcome to Acme Web Sulutions</h1>
           consectetur, adipisci velit...
         </div>
         <div class="button-container">
-          <button class="button-header">
+          <button
```
Comment (takrami) [inline index.html:25]: great job with using aria-label :+1:  i'm wondering can we use it for more elements or would be unnecessary?

**Review by takrami — COMMENTED** [review]
taking accessibility to consideration is really a huge improvement :100:

File: index.html [hunk 4]
```diff
@@ -138,5 +144,19 @@ <h2 class="fifth-section-title">
...
+       if (currentlyLightTheme){
+       main.className = "overrideLight";
+       currentlyLightTheme = false;
+    }
+      else {
+       main.className = "overrideDark";
+       currentlyLightTheme = true;
+    }
+  }
+    </script>
```
Comment (blacksrc) [inline index.html]: Please fix the indentation :grin:

Comment (blacksrc) [reply index.html]: Plus, double check if this `window.matchMedia("prefers-color-schene: light")` is returning the correct value that is required for the rest of the code.

File: index.html (hunk 1 above)
Comment (hedita) [reply index.html]: Oh!You are right.

File: index.html (hunk 3 above)
Comment (hedita) [reply index.html:25]: Can you tell which element for example?

File: index.html (hunk 4 above)
Comment (hedita) [reply index.html]: The value is main,which refrence to body,am I wrong?

Comment (blacksrc) [reply index.html]: I mean for the first time. Does this return the correct value? I heard that you had a bug for the first click. My guess is, that's not returning the default theme for the first time.

File: index.html [hunk 5]
```diff
@@ -22,12 +22,17 @@ <h1 class="title-header">Welcome to Acme Web Sulutions</h1>
...
+            aria-label="jump to seconed section"
+            tabindex="0"
+            class="button-header"
+          >
             <a href="#jump-seconed-section">READ MORE</a>.
           </button>
         </div>
       </div>
     </header>
+    <label>Enable Dark Mode <input onclick="toggle()" id="input" type="checkbox" />
```
Comment (blacksrc) [inline index.html]: `<label>` cannot see the closing tag :thinking:

File: index.html (hunk 3 above)
Comment (takrami) [reply index.html:25]: just was a question. if according to the course there is no more element that's totally fine :)

**Review by takrami — APPROVED** [review]
well done with this pr :100:

**Pull request merged**